### ISSUE TITLE
Accessibility:  U-M Users Multiple Sections header tag

### ISSUE THREAD

**Issue by ktowneUM** [tl-its-umich-edu/canvas-course-manager-next#642]
Based on the [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

Severity: Major

Feature: U-M Users Multiple Sections

Issue notes:
File: filename.csv" should be p not h5

Criterion:
2.4.10 Section Headings
